examples/text_to_image/train_text_to_image_lora_sdxl.py

```diff
@@ -560,7 +566,12 @@ def main(args):
     )
 
     # Load scheduler and models
-    noise_scheduler = DDPMScheduler.from_pretrained(args.pretrained_model_name_or_path, subfolder="scheduler")
+    noise_scheduler = DDPMScheduler.from_pretrained(
+        args.pretrained_model_name_or_path,
+        subfolder="scheduler",
+        rescale_betas_zero_snr=args.scale_scheduler,
+        timestep_spacing="trailing" if args.scale_scheduler else None,
+    )
```

Comment:
If it's used with the Min-SNR strategy, this might lead to numerical instabilities: 

https://github.com/huggingface/diffusers/blob/a7de96505b8c71a626c6fa45ff3cf85e42db4960/examples/text_to_image/train_text_to_image_lora_sdxl.py#L666

This is how we deal with it [here](https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/train_text_to_image_sdxl.py#L557C1-L568C10). Cc: @bghira.